# Initialize parameters of the Object (RVIZ)
object_pos: [0.0, 0.0, 0.0]
object_visible: False

# Initialize home parameters of the robot arm
# UR5: [1.57, -1.57, 1.57, -1.57, -1.57, 0.0]
# ABB IRB 1200: [0.0, 0.0, 0.0, 0.0, 1.57, 0.0]
# FANUC CR7IA [0.0, 0.0, 0.0, 0.0, -1.57, 0.0]
robot_home_param: [0.0, 0.0, 0.0, 0.0, 1.57, 0.0]